tests/testdata/cluster-role-binding_webhook-validation.yaml

```diff
@@ -0,0 +1,12 @@
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: cass-operator
+subjects:
+- kind: ServiceAccount
+  name: cass-operator
+  namespace: test-webhook-validation 
```

Comment:
@jimdickinson , actually I think I was mistaken in my namespace comment in the other PR.  I had to specify a value for namespace here to make things work.  It didn't accept "".  Therefore I don't think we can have a generic setting for namespace.  We have to rely on helm or some other process to set a value.

Reply:
ok, this is actually good news, we just need a step to deal with it, and @sandoichi is working on that now